examples/coprocessor_java/common/build.gradle

```diff
@@ -0,0 +1,12 @@
+
+apply plugin: 'eclipse'
+apply plugin: 'java'
```

Comment:
Please remove these plugins, the java plugin is defined in the root project, and the eclipse one is unnecessary (the idea one is already applied at the root)

Reply:
Done.